### PR TITLE
Live: push scope to demultiplex incoming data to stream channels

### PR DESCRIPTION
**Pull request by FZambia**
**What this PR does / why we need it**:

This is what HTTP push endpoint does but over WS.

### CODE REVIEW

File: pkg/services/live/demultiplexer.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	return s, nil
+}
+
+func (s *Demultiplexer) OnSubscribe(_ context.Context, _ *models.SignedInUser, e models.SubscribeEvent) (models.SubscribeReply, backend.SubscribeStreamStatus, error) {
+	reply := models.SubscribeReply{}
+	return reply, backend.SubscribeStreamStatusOK, nil
+}
+
+func (s *Demultiplexer) OnPublish(_ context.Context, _ *models.SignedInUser, evt models.PublishEvent) (models.PublishReply, backend.PublishStreamStatus, error) {
+	stream, err := s.managedStreamRunner.GetOrCreateStream("archer")
```
Comment (ryantxu) [inline pkg/services/live/demultiplexer.go:34]: should be somethign like
```suggestion
	stream, err := s.managedStreamRunner.GetOrCreateStream("$namespace$")
```

File: pkg/services/live/demultiplexer.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	}
+}
+
+func (s *Demultiplexer) GetHandlerForPath(_ string) (models.ChannelHandler, error) {
+	return s, nil
+}
+
+func (s *Demultiplexer) OnSubscribe(_ context.Context, _ *models.SignedInUser, e models.SubscribeEvent) (models.SubscribeReply, backend.SubscribeStreamStatus, error) {
+	reply := models.SubscribeReply{}
+	return reply, backend.SubscribeStreamStatusOK, nil
```
Comment (ryantxu) [inline pkg/services/live/demultiplexer.go:30]: should we throw an error for now?  I don't think we want to support/encourage subscribing to the push endpoint.  It is really a utility to match the HTTP endpoint

File: pkg/services/live/demultiplexer.go (hunk 1 above)
Comment (FZambia) [reply pkg/services/live/demultiplexer.go]: fixed

File: pkg/services/live/demultiplexer.go (hunk 2 above)
Comment (FZambia) [reply pkg/services/live/demultiplexer.go]: done

File: pkg/services/live/scope.go [hunk 3]
```diff
@@ -9,4 +9,6 @@ const (
 	ScopeDatasource = "ds"
 	// ScopeStream is a managed data frame stream
 	ScopeStream = "stream"
+	// ScopePush is a managed data frame demultiplexer.
```
Comment (ryantxu) [inline pkg/services/live/scope.go:12]:
```suggestion
	// ScopePush allows sending data into managed streams.
```

**Review by ryantxu — APPROVED** [review]
works for me

**Pull request merged**